Content.Client/Overlays/ColorTintOverlay.cs

```diff
@@ -22,7 +23,7 @@ public sealed class ColorTintOverlay : Overlay
     /// <summary>
     ///     The color to tint the screen to as RGB on a scale of 0-1.
     /// </summary>
-    public Vector3? TintColor = null;
+    public Robust.Shared.Maths.Vector3? TintColor = null;
```

Comment:
Does it specifically has to be Robust Vector3? I don't think so honestly.

Reply:
yes